governance/polls/Ratification Poll for Methodology and Review Process for Structured Finance Transactions (MIP 67) - April 11, 2022.md

```diff
@@ -0,0 +1,59 @@
+---
+title: Ratification Poll for Methodology and Review Process for Structured Finance Transactions (MIP67) - April 11, 2022
+summary: MIP67 intends to formalise the methodology and high-level process to evaluate structured finance transactions made under the MIP6 application process.
+discussion_link: https://forum.makerdao.com/t/mip67-methodology-and-review-process-for-structured-finance-transactions/13737
+vote_type: Plurality Voting
+categories:
+   - MIPs
+options:
+   0: Abstain
+   1: Yes
+   2: No
+start_date: 2022-04-11T16:00:00
+end_date: 2022-04-25T16:00:00
+---
+# Ratification Poll for Methodology and Review Process for Structured Finance Transactions (MIP67) - April 11, 2022
+
+The Governance Facilitators have placed a ratification poll into the [voting system](https://vote.makerdao.com/polling) as part of the responsibilities defined in [MIP51](https://mips.makerdao.com/mips/details/MIP51). This Governance [Poll](https://community-development.makerdao.com/en/learn/governance/on-chain-gov) will be active for fourteen days beginning on Monday, April 11 at 16:00 UTC.
+
+**This is a binary vote.**
+- **You may vote for a single option.**
+- **You should vote for the option which you prefer.**
+- **If you would accept either option, you should vote 'Abstain'.**
+
+## Review
+
+The community may vote in this poll to express support or opposition to MIP67 being accepted and implemented in the Maker Protocol.
+
+A brief summary of this proposal has been provided by the MIP Author and is shown below:
+
+*MIP67 intends to (1) formally approve the Methodology for Review of MIP6 Structured Finance Transactions 17 published by the Real-World Finance Core Unit, and (2) introduce a high-level process that each MIP6-compliant application should undergo in order to receive, eventually, a positive endorsement from the relevant real-world CU stakeholder groups. E.g. RWF-001, SES-incubated Lending Oversight team, future CUs.*
+
+Please review the links below to inform your position on this proposal before voting.
+* [Canonical Proposal Version](https://github.com/makerdao/mips/commit/7ea0b7ab8198bb5ff018a15f57e9766427ace6ca)
```

Comment:
Use this link: https://github.com/makerdao/mips/blob/7ea0b7ab8198bb5ff018a15f57e9766427ace6ca/MIP67/MIP67.md